segmentation/include/pcl/segmentation/euclidean_cluster_comparator.h

```diff
@@ -162,22 +124,21 @@ namespace pcl
         exclude_labels_ = boost::make_shared<std::vector<bool> >(exclude_labels);
```

Comment:
This looks weird, I see no point for copying excludes to a shared pointer and they should be passed as const